### ISSUE TITLE
Error: Unknown OS

### ISSUE THREAD

**Issue by H4Security** [avast-tl/retdec#10]
when I try to execute decompile.sh I got this message "Error: Unknown OS"
I try it on windows 7 64

**Comment by s3rvac** [comment]
Thank you for the report. Which terminal emulator and shell do you use to run `decompile.sh`? What is the output from `uname -s`?

**Comment by H4Security** [comment]
I was using bash to run decompile.sh 
"uname -s" result is "WindowsNT"
Thanks

**Comment by s3rvac** [comment]
Fixed in b19112dc8dc1eb5f48aa51c13320776ff1960c1e. The fix will be included in the next release.

Until then, as a workaround, remove the following code from `$RETDEC_INSTALL_DIR/bin/config.sh`:
```
SYS="$(get_SYS)"
if [ "$SYS" != "win" -a "$SYS" != "linux" ]; then
   echo "Error: Unknown OS" >&2
   exit 1
fi
```
